age/Cargo.toml

```diff
@@ -21,7 +21,8 @@ age-core = { version = "0.3.1", path = "../age-core" }
 base64 = "0.11"
 
 # - ChaCha20-Poly1305 from RFC 7539
-chacha20poly1305 = "0.3"
+c2-chacha = "0.2"
+chacha20poly1305 = "0.4"
```

Comment:
FYI, you should be able to do:

```toml
chacha20poly1305 = { version = "0.4", default = false }
```

...which will eliminate the `chacha20` dependency, which is redundant if you're using `c2-chacha`

/cc @cryptocorrosion